maps/tether/tether_jobs.dm

```diff
@@ -0,0 +1,184 @@
+///////////////////////////////////
+//// Talon Jobs
+/datum/department/talon
+	name = DEPARTMENT_TALON
+	short_name = "Talon"
+	color = "#888888"
+	sorting_order = -2
+	assignable = FALSE
+	visible = FALSE
+
+/datum/job/talon/captain
+	title = "Talon Captain"
+	departments_managed = list(DEPARTMENT_TALON)
+	job_description = "The captain's job is to generate profit through trade or other means such as salvage or even privateering."
```

Comment:
Captain does not have 'supervisors' but I believe current code implies ALL jobs must have a supervisors string because of how welcome message is constructed. Could even be something jokey, similar to Mime/Clown's supervisors, but it has to have it in order for message not to botch itself.

Reply:
Added